src/sentry/auth/providers/saml2.py

```diff
@@ -14,7 +14,9 @@
 
 try:
     from onelogin.saml2.auth import OneLogin_Saml2_Auth, OneLogin_Saml2_Settings
+    HAS_SAML2 = True
 except ImportError:
+    HAS_SAML2 = False
 
     def OneLogin_Saml2_Auth(*args, **kwargs):
```

Comment:
unrelated, but i was thinking maybe we hsould start a pattern of "if feature: from _module_name import *" and put the actual code elsewhere (to avoid crazy indentation)